solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -728,7 +730,7 @@ contract RandomBeacon is Ownable {
 
         if (slashingAmount > 0) {
             address[] memory groupMembers = sortitionPool.getIDOperators(
-                group.members
+                members
```

Comment:
Please remember about checks if provided members list corresponds to the hash stored on DKG result approval.

Reply:
Sure, added checks